drivers/qsc/q_sys_camera.cr

```diff
@@ -0,0 +1,92 @@
+# encoding: ASCII-8BIT
+# frozen_string_literal: true
```

Comment:
forgot to mention this the other day @rdiazx  but I don't think these lines are needed in PlaceOS, can you confirm @stakach ?

Reply:
Yeah not needed, all strings are UTF8 in crystal
That was a ruby hack to get around windows encoding leaking in if people edited files on windows